pkgs/by-name/on/onnxruntime/package.nix

```diff
@@ -268,7 +263,8 @@ effectiveStdenv.mkDerivation rec {
 
   passthru = {
     inherit cudaSupport cudaPackages; # for the python module
-    protobuf = protobuf_21;
+    # protobuf = protobuf_21;
```

Comment:
Im guessing you arent done here, otherwise is there a reason why you wanted to comment out rather than remove this line ?